snippets/rust.json

```diff
@@ -20,6 +20,10 @@
       "prefix": "for",
       "body": "for ${1:pat} in ${2:expr} {\n\t$3\n}"
     },
+    "format": {
+      "prefix": "format",
+      "body": "format!(\"${1:\\{\\}}\", ${2:value})$0"
```

Comment:
To keep consistency with existing snippets, prefer `$3` as the last tab stop instead of the _(special?)_ `$0` here, since the last stop should be at the end anyway.